tests/integration/test_terraform.py

```diff
@@ -12,23 +13,33 @@
 LAMBDA_RUNTIME = 'dotnetcore2.0'
 LAMBDA_ROLE = 'arn:aws:iam::000000000000:role/iam_for_lambda'
 
+INIT_LOCK = threading.RLock()
 
-# TODO: test temporarily disabled
-# class TestTerraform(unittest.TestCase):
-class TemporarilyDisabled:
+
+class TestTerraform(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        base_dir = os.path.join(os.path.dirname(__file__), 'terraform')
-        if not os.path.exists(os.path.join(base_dir, '.terraform')):
-            run('cd %s; terraform init -input=false' % base_dir)
-        run('cd %s; terraform plan -out=tfplan -input=false' % (base_dir))
-        run('cd %s; terraform apply -input=false tfplan' % (base_dir))
+        with(INIT_LOCK):
+            run('cd %s; terraform apply -input=false tfplan' % (TestTerraform.get_base_dir()))
```

Comment:
nit: `TestTerraform.get_base_dir()` could be replaced with `cls.get_base_dir()` (slightly shorter)